app/services/queries/create_query_service.rb

```diff
@@ -31,7 +31,11 @@
 require_relative 'query_service'
 
 class CreateQueryService < QueryService
-  self.contract = Queries::CreateContract
+
+  def initialize(**args)
+    super(**args)
```

Comment:
Just calling
```
super
```
would call super with all the arguments provided. Just mentioning because I have seen you list the parameters on all the super calls which is not necessary if super`s signature is the same as the calling methods`.